### DISCUSSION TITLE
LoRa_APRS_iGate in loop

### DISCUSSION THREAD

**Discussion by beppelpo** [lora-aprs/LoRa_APRS_iGate#306] [General]
Hi Peter… I use a LilYgo LoRa32 V2.1 433MHz ESP32 LoRa as I-Gate with your software. All ok but it doesn't transmit the beacon at 433.775Mhz, only on aprs.is via network. If on "is-cfg.json" under LORA I set "tx_enable": true when restarting Lilygo goes into a loop, it tries to restart but continuously returns to the initial message as in the attached video. 73's de beppe i3lpo

https://github.com/lora-aprs/LoRa_APRS_iGate/assets/100496682/7c21ed5e-d287-4739-9180-8bfc3674dedd

**Comment by peterus** [comment]
Hi beppe!
issue is known and should be fixed on mainline!
there was also a RX issue which should be fixed exactly now on mainline.
ticket is here for more details: https://github.com/lora-aprs/LoRa_APRS_iGate/issues/296
please provide feedback in the ticket, thanks!

**Reply by beppelpo** [reply]
TNX Peter
